.github/meta.yaml

```diff
@@ -28,7 +28,7 @@ outputs:
         - pandas >=1.5.0
         - dask >=2022.2.0,!=2022.10.1
         - scipy >=1.5.0
-        - scikit-learn >=1.2.1
+        - scikit-learn >=1.2.2
```

Comment:
can we add a release note mentioning this and the holidays dependency changes?

Reply:
good call!